scripts/cleanup-sam-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# A script for cleaning up a user's Sam resources, such as those not cleaned
+# up after testing. This will attempt to delete all controlled (both shared and
+# private) resources owned by a user in Sam, as well as all their workspaces.
+#
+# This script will attempt to delete all of the resources and workspaces a user
+# has any access to, including ones the user does not have permission to delete
+# (such as those owned by other test users, where the specified user is only
+# a reader). This means some delete calls will fail with a 403 error and the
+# message "You may not perform any of [DELETE] on
+# {resource type}/{resource id}". This is expected and can be ignored.
+#
+# Input:
+#   access_token: An access token used for calls to Sam. This script will delete
+#   all controlled resources and workspaces in Sam that the user owns, so be
+#   sure this belongs to the correct user.
+
+access_token=$1
```

Comment:
It'd be neat to give a confirmation option, like "This will delete 2324 resources. Continue?".

Reply:
I like that idea, thanks!